src/scope/core/pipelines/schema.py

```diff
@@ -154,40 +287,105 @@ class StreamDiffusionV2Config(BasePipelineConfig):
+                input_size=None,
+                denoising_steps=[1000, 750],
+                default_prompts=["A dog in the grass looking around, photorealistic"],
+            ),
+            "video": ModeDefaults(
+                # Video mode: requires input frames, noise controls active
+                noise_scale=0.7,
+                noise_controller=True,
+                input_size=4,
+                default_prompts=["A dog in the grass looking around, photorealistic"],
+            ),
+        }
+
 
 class KreaRealtimeVideoConfig(BasePipelineConfig):
-    """Configuration for Krea Realtime Video pipeline."""
+    """Configuration for Krea Realtime Video pipeline.
+
+    Krea supports both text-to-video and video-to-video modes.
+    Default mode is text (T2V was the original training focus).
+    """
 
     pipeline_id: ClassVar[str] = "krea-realtime-video"
     pipeline_name: ClassVar[str] = "Krea Realtime Video"
     pipeline_description: ClassVar[str] = (
         "High-quality real-time video generation with 14B model"
     )
 
-    # Krea defaults
-    height: int = Field(default=480, ge=1, description="Output height in pixels")
-    width: int = Field(default=832, ge=1, description="Output width in pixels")
+    # Mode support
+    supported_modes: ClassVar[list[InputMode]] = ["text", "video"]
+    default_mode: ClassVar[InputMode] = "text"
+
+    # Krea defaults (text mode baseline) - distinct from LongLive (320x576)
+    height: int = Field(default=320, ge=1, description="Output height in pixels")
+    width: int = Field(default=576, ge=1, description="Output width in pixels")
     denoising_steps: list[int] | None = Field(
         default=[1000, 750, 500, 250],
         description="Denoising step schedule for progressive generation",
     )
-    # Video mode default (previously hardcoded in pipeline)
+    # noise_scale is None by default (text mode), overridden in video mode
     noise_scale: Annotated[float, Field(ge=0.0, le=1.0)] | None = Field(
-        default=0.7,
+        default=None,
         description="Amount of noise to add during video generation (video mode only)",
     )
 
+    @classmethod
+    def get_mode_defaults(cls) -> dict[InputMode, ModeDefaults]:
+        """Krea mode-specific defaults."""
+        return {
+            "text": ModeDefaults(
+                # Text mode: no video input, no noise controls
+                noise_scale=None,
+                noise_controller=None,
+                input_size=None,
+                default_prompts=[
+                    "A 3D animated scene. A **panda** walks along a path towards the camera in a park on a spring day."
+                ],
+            ),
+            "video": ModeDefaults(
+                # Video mode: requires input frames, noise controls active
+                height=256,
+                width=256,
+                noise_scale=0.7,
+                noise_controller=True,
+                input_size=4,
```

Comment:
This doesn't seem right? If krea uses num_frame_per_block = 3 then with VAE temporal factor = 4 that should be an input size of 12?